Utilities/OpenCorePackage/Resource/UEFI-DUET/Resources/en.lproj/Readme.rtfd/TXT.rtf

```diff
@@ -0,0 +1,60 @@
+{\rtf1\ansi\ansicpg1252\cocoartf2638
+\cocoatextscaling0\cocoaplatform0{\fonttbl\f0\fswiss\fcharset0 Helvetica;\f1\fnil\fcharset128 HiraginoSans-W3;}
+{\colortbl;\red255\green255\blue255;\red0\green0\blue0;\red255\green255\blue255;\red0\green0\blue0;
+\red255\green255\blue255;\red255\green255\blue255;\red27\green128\blue255;}
+{\*\expandedcolortbl;;\cssrgb\c0\c0\c0\c87059;\cssrgb\c100000\c100000\c100000\c0;\cssrgb\c0\c1\c1;
+\cssrgb\c100000\c100000\c99985\c0;\cssrgb\c100000\c100000\c99956\c0;\cssrgb\c11095\c58865\c100000;}
+\margl1440\margr1440\vieww11520\viewh8400\viewkind0
+\deftab720
+\pard\pardeftab720\qc\partightenfactor0
+
+\f0\fs28 \cf2 \cb3 \expnd0\expndtw0\kerning0
+This program will replace Opencore in the EFI partition.\
+Apple folder will not be replaced if it exists.\
```

Comment:
Rather than mentioning the APPLE folder explicitly, a guarantee should be given for any unrelated folder (i.e., mention the modified folders explicitly and explicitly state any other will be untouched).